docs/guides/serverless-api-with-nextjs.md

```diff
@@ -0,0 +1,512 @@
+## Overview
+
+This guide shows you how to create a simple To-do list in Nextjs with an API backed by serverless functions. It also uses collections for data persistence.
```

Comment:
Can we change to call out that we are building with Nextjs and Nitric.

E.g. change serverless to Nitric functions